composer.json

```diff
@@ -14,9 +14,10 @@
         }
     ],
     "require": {
-        "php": "^5.6 || ^7.0",
+        "php": "^7.0",
```

Comment:
will work on how the compat with old php can/should look like, after I get it working on current php versions

Reply:
You might want to see #4810.

Reply:
If we don't want PHP 5, then why do we even need a parallel linter? We can use the tokenising linter on PHP 7?

Reply:
> why do we even need a parallel linter? We can use the tokenising linter on PHP 7?

I had the theory that tokenized linting in parallel process is faster then doing it in a single process